src/TrackHists.cxx

```diff
@@ -30,6 +30,12 @@ TrackHists::TrackHists() {
   h_nhit3 =
       new TH1F("reco_nhit_outer", ";Outer tracker track hits; Tracks [/hit]",
                20, -0.5, 19.5);
+  if (effi) {
+    h_effpt = new TEfficiency("fake_pt_vs_eff", 
+      "Fake Track Rate vs pT;Reconstructed pT [GeV]; Efficiency", 50, 0, 12);
+    h_effeta = new TEfficiency("fake_eta_vs_eff", 
+      "Fake Track Rate vs Eta;Reconstructed eta; Efficiency", 50, -4, 4);
```

Comment:
Make plots vs theta instead to align it as all other plots currently use that unit.